application/libraries/REST_Controller.php

```diff
@@ -1,7 +1,12 @@
 <?php
-
 defined('BASEPATH') OR exit('No direct script access allowed');
 
+require APPPATH . '/libraries/jwt/JWT.php';
+require APPPATH . '/libraries/jwt/BeforeValidException.php';
```

Comment:
you are right, i am afraid to use composer, some developers prefer to include the library.